tests/ci/docker_manifests_merge.py

```diff
@@ -198,6 +276,9 @@ def main():
             if test_result != "OK":
                 status = "failure"
 
+    # changed_images now contains all the images that are changed in this PR. Let's find the latest tag for the images that are not changed.
+    changed_images = enrich_images(changed_images)
```

Comment:
In case of failure on the CH side, we need to fail it over to existing logic with `:latest`

1. We need to have the same exceptions class as in vendored shared lambdas https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/lambda_shared_package/lambda_shared/__init__.py#L36-L41, it's surprising I didn't tweak the main module accordingly
2. Here we need `try; except CHException` to continue with existing changed images.

Reply:
CHHelper already does 5 retries in `_select_and_get_json_each_row`. If it fails, it fails ¯\_(ツ)_/¯

The previous version was good enough, no needs to make 50 retries (10 * 5)

Reply:
Ahh, it is true.